### PR TITLE
Add ground spawns to discovered items feature.

### PR DESCRIPTION
**Pull request by noudess**
Added code to allow ground spawns to be discovered items.

### CODE REVIEW

**Review by Kinglykrab — APPROVED** [review]

File: zone/object.cpp [hunk 1]
```diff
@@ -535,6 +535,13 @@ bool Object::HandleClick(Client* sender, const ClickObject_Struct* click_object)
 			sender->PutItemInInventory(EQ::invslot::slotCursor, *m_inst, false);
 			sender->SendItemPacket(EQ::invslot::slotCursor, m_inst, ItemPacketTrade);
 
+			// Could be an undiscovered ground_spawn
+			if (m_ground_spawn && (RuleB(Character, EnableDiscoveredItems)))
+			{
+				if (!sender->GetGM() && !sender->IsDiscovered(item->ID))
```
Comment (Kinglykrab) [inline zone/object.cpp:541]: Brackets.

Comment (noudess) [reply zone/object.cpp:541]: Want a set around the one liner, or you want different indenting?

Comment (noudess) [reply zone/object.cpp:541]: okies

Comment (nytmyr) [reply zone/object.cpp:541]: I'm curious about this as well, I have noticed the recent commits have been bracketing one liners whereas most original code seems to not bracket.

I personally like no brackets for legibility but not opposed either way.

However, merge conflicts for custom code can become problematic if brackets are added to areas where there previously weren't. I had a ton of sections that threw errors where auto merges weren't adding either the first or second brackets if there were surrounding custom code. Easy fixes and it's only a custom code problem not related to master but just something I noticed.

Comment (fryguy503) [reply zone/object.cpp:541]: Oneliners should always be bracketed due to prior history of issues related to implied brackets in complex code not being noticed. All PR's in the last roughly 2 years have been much more strict on overall formatting to keep a clean codebase.

Comment (Kinglykrab) [reply zone/object.cpp:541]: > Oneliners should always be bracketed due to prior history of issues related to implied brackets in complex code not being noticed. All PR's in the last roughly 2 years have been much more strict on overall formatting to keep a clean codebase.

This. We haven’t always had the standards and leads to some issues over time where scope isn’t explicit. We’re trying to be cleaner about it now.

**Review by Aeadoin — APPROVED** [review]

**Pull request merged**